src/course-outline/card-header/CardHeader.jsx

```diff
@@ -50,16 +55,18 @@ const CardHeader = ({
   discussionEnabled,
   discussionsSettings,
   parentInfo,
-  tagsCount,
 }) => {
   const intl = useIntl();
   const [searchParams] = useSearchParams();
   const [titleValue, setTitleValue] = useState(title);
   const cardHeaderRef = useRef(null);
+  const [isManageTagsDrawerOpen, openManageTagsDrawer, closeManageTagsDrawer] = useToggle(false);
 
   const isDisabledPublish = (status === ITEM_BADGE_STATUS.live
     || status === ITEM_BADGE_STATUS.publishedNotLive) && !hasChanges;
 
+  const { data: contentTagCount } = useContentTagsCount(cardId);
```

Comment:
@rpenido Will we follow the method of making a call on each card? I'm still leaning toward the approach of [fetch tag counts of the entire course](https://github.com/open-craft/frontend-app-course-authoring/pull/30#discussion_r1516775926).

Reply:
The current code fetches tags for the entire course, but the hook returns the value only for the requested `contentId`.

See:

https://github.com/openedx/frontend-app-course-authoring/blob/d1bad89bf1de76d264190eea5ff23953b500789d/src/course-outline/data/apiHooks.jsx#L11-L23

Reply:
Ohh, I understand, thanks :+1: